ui-spacetimechart/src/utils/scales.ts

```diff
@@ -107,17 +107,26 @@ export function spaceScalesToBinaryTree(
  *
  * Also, if the position is lower than the tree's min (tree.from), then the first leaf is returned
  * and if it is higher than the max (tree.to), the last leaf is returned.
+ *
+ * Finally, if pickLast is truthy, then it returns the last leaf node that contains that position
+ * instead of the first one. It is very important when there are flat sections (
```

Comment:
My comment isn't very helpful, but there's an extra parenthesis. 
```suggestion
 * instead of the first one. It is very important when there are flat sections
```

Reply:
Thanks! Fixed.